From 0743381573e4e4c50039aef33cf7404baa340c3d Mon Sep 17 00:00:00 2001
From: Olivier Fourdan <ofourdan@redhat.com>
Date: Tue, 10 Mar 2020 14:44:15 +0100
Subject: [PATCH 14/48] window/x11: Rename
 `meta_window_x11_buffer_rect_to_frame_rect`

To keep consistent and avoid confusion, rename the function:
    `meta_window_x11_buffer_rect_to_frame_rect()`
to:
    `meta_window_x11_surface_rect_to_frame_rect()`

As this function doesn't deal with the `window->buffer_rect` at all.

https://gitlab.gnome.org/GNOME/mutter/merge_requests/1091
---
 src/compositor/meta-window-actor-x11.c | 2 +-
 src/x11/window-x11.c                   | 8 ++++----
 src/x11/window-x11.h                   | 4 ++--
 3 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/compositor/meta-window-actor-x11.c b/src/compositor/meta-window-actor-x11.c
index 851df7e57..a8d137316 100644
--- a/src/compositor/meta-window-actor-x11.c
+++ b/src/compositor/meta-window-actor-x11.c
@@ -929,7 +929,7 @@ build_and_scan_frame_mask (MetaWindowActorX11    *actor_x11,
        */
       if (meta_window_x11_always_update_shape (window))
         {
-          meta_window_x11_buffer_rect_to_frame_rect (window, &rect, &frame_rect);
+          meta_window_x11_surface_rect_to_frame_rect (window, &rect, &frame_rect);
           get_client_area_rect_from_texture (actor_x11, stex, &client_area);
         }
       else
diff --git a/src/x11/window-x11.c b/src/x11/window-x11.c
index 49f7f1526..ed56bca5f 100644
--- a/src/x11/window-x11.c
+++ b/src/x11/window-x11.c
@@ -4075,9 +4075,9 @@ meta_window_x11_always_update_shape (MetaWindow *window)
 }
 
 void
-meta_window_x11_buffer_rect_to_frame_rect (MetaWindow    *window,
-                                           MetaRectangle *buffer_rect,
-                                           MetaRectangle *frame_rect)
+meta_window_x11_surface_rect_to_frame_rect (MetaWindow    *window,
+                                            MetaRectangle *surface_rect,
+                                            MetaRectangle *frame_rect)
 
 {
   MetaFrameBorders borders;
@@ -4086,7 +4086,7 @@ meta_window_x11_buffer_rect_to_frame_rect (MetaWindow    *window,
 
   meta_frame_calc_borders (window->frame, &borders);
 
-  *frame_rect = *buffer_rect;
+  *frame_rect = *surface_rect;
   frame_rect->x += borders.invisible.left;
   frame_rect->y += borders.invisible.top;
   frame_rect->width -= borders.invisible.left + borders.invisible.right;
diff --git a/src/x11/window-x11.h b/src/x11/window-x11.h
index 7f4f96f6c..d41d0d156 100644
--- a/src/x11/window-x11.h
+++ b/src/x11/window-x11.h
@@ -89,8 +89,8 @@ void     meta_window_x11_set_thaw_after_paint    (MetaWindow *window,
 gboolean meta_window_x11_should_thaw_after_paint (MetaWindow *window);
 gboolean meta_window_x11_always_update_shape     (MetaWindow *window);
 
-void meta_window_x11_buffer_rect_to_frame_rect   (MetaWindow    *window,
-                                                  MetaRectangle *buffer_rect,
+void meta_window_x11_surface_rect_to_frame_rect  (MetaWindow    *window,
+                                                  MetaRectangle *surface_rect,
                                                   MetaRectangle *frame_rect);
 void meta_window_x11_surface_rect_to_client_rect (MetaWindow    *window,
                                                   MetaRectangle *surface_rect,
-- 
2.26.0.rc2

